.github/workflows/docker-image.yml

```
@@ -108,7 +108,7 @@ jobs:
     - name: Run tests with coverage
       working-directory: ./PerfumeTracker.xTests
       env:
-       ConnectionStrings__DefaultConnection: ${{ secrets.POSTGRES_CONNSTR }}
+       ConnectionStrings__PerfumeTracker: ${{ secrets.POSTGRES_CONNSTR }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 7

(indentation)